include/lemlib/movements/boomerang.hpp

```diff
@@ -38,25 +38,25 @@ class Boomerang : public Movement {
          * @brief Update the movement
          *
          * @note This function should be called in a loop
-         * @note Units are in inches and radians. Angle 0 is right, and increases counterclockwise
+         * @note Angle 0 is right, and increases counterclockwise
          *
          * @param pose the position of the chassis
          *
          * @return std::pair<int, int> left and right motor power respectively. 128 means movement is done
          */
         std::pair<int, int> update(Pose pose) override;
     private:
-        FAPID angularPID;
-        FAPID linearPID;
+        FAPID<Angle> angularPID;
+        FAPID<Length> linearPID;
         Pose target;
-        Pose prevPose = Pose(0, 0, 0);
+        Pose prevPose = Pose(0_in, 0_m, 0_rad);
```

Comment:
```suggestion
        Pose prevPose = Pose(0_m, 0_m, 0_rad);
```
Internal length units should be consistent